console/console-init/ui/src/modules/address-detail/components/AddressDetailHeader/AddressDetailHeader.tsx

```diff
@@ -25,6 +25,8 @@ export interface IAddressDetailHeaderProps {
   topic?: string | null;
   name: string;
   plan: string;
+  Deadletters?: string | null;
```

Comment:
please revisit this name. All props will be camel cased. like `deadLetter` and `expiryQueue`